app/src/main/java/com/nononsenseapps/feeder/model/FeedParser.kt

```diff
@@ -286,6 +443,93 @@ class FeedParser(override val di: DI) : DIAware {
     }
 }
 
+private fun List<Event>.mapToFeed(
+    nostrUri: String,
+    authorName: String,
+    imageUrl: String,
+): ParsedFeed {
+    val description = "Nostr articles by $authorName"
```

Comment:
Is this string user visible? If so it should not be hardcoded in english

Reply:
Fixed.